cdap-common/src/test/java/co/cask/cdap/common/conf/ArtifactConfigReaderTest.java

```diff
@@ -108,6 +108,16 @@ public void testMissingPluginFields() throws IOException, InvalidArtifactExcepti
     configReader.read(Id.Namespace.SYSTEM, configFile);
   }
 
+  @Test
+  public void testMissingPlugins() throws IOException, InvalidArtifactException {
+    // plugin JSON files written by cdap-maven-plugin does not have plugins information
+    File configFile = new File(tmpFolder.newFolder(), "r1-1.0.0.json");
+    try (BufferedWriter writer = Files.newWriter(configFile, Charsets.UTF_8)) {
+      writer.write("{ \"parents\": [], \"properties\": {} }");
```

Comment:
create an ArtifactConfig and serialize it instead of writing the string directly.